Workspaces/Project/ProjectWorkspace.cs

```diff
@@ -98,6 +99,10 @@ public override void Setup()
 			connectInterfaces(mb);
 		}
 
+		var zoomTooltip = sliderObject.GetComponentInChildren<Tooltip>();
```

Comment:
The problem is that ITooltip doesn't provide a setter on tooltipText. We can discuss whether this needs to change.